frame/composable-traits/src/vamm.rs

```diff
@@ -78,7 +78,7 @@ pub trait Vamm {
 }
 
 /// Specify a common encapsulation layer for the [`create`](Vamm::create) function.
-#[derive(Clone, Debug, Default)]
+#[derive(Clone, Debug, Decode, Default, Encode, MaxEncodedLen, PartialEq, TypeInfo)]
```

Comment:
@Cardosaum I noticed that these were necessary for building the runtime. How did you get around it in your tests?

Reply:
:thinking: I think it might be because I specify these constraints in the types itself on `lib.rs`?